src/libpriv/rpmostree-passwd-util.c

```diff
@@ -409,6 +410,8 @@ rpmostree_check_passwd_groups (gboolean         passwd,
             goto out;
           
           old_path = g_file_resolve_relative_path (root, commit_filepath);
+          if (!g_file_load_contents (old_path, cancellable, &old_contents, NULL, NULL, error))
+            goto out;
```

Comment:
That is an excellent question :crystal_ball: !  That's actually what I did at first, and was very confused why things were breaking, only to later realize that in the (default) "previous" case we're actually extracting the content from OSTree's `OstreeRepoFile*`, and using `gs_file_get_path_cached()` on that gave the normal `/usr/lib/passwd` which was ENOENT *only* because I happened to be running inside a Docker container.  Otherwise on my host system it would have done the wrong :fire: thing.